adapter.js

```diff
@@ -25,14 +25,14 @@
       }
     },
     exception: function(reason) {
-      var error, d;
-
-      error = new Error(reason);
+      if (!reason instanceof Error) {
```

Comment:
I think this expression is always true. It should probably be `if (!(reason instanceof Error))`.